src/components/user/CreateTargetForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { PureComponent } from 'react';
+import { func, string, bool, array } from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import {
+  injectIntl,
+  intlShape,
+  defineMessages,
+  FormattedMessage
+} from 'react-intl';
+
+import Loading from 'components/common/Loading';
+import Input from 'components/common/Input';
+import CustomSelect from 'components/common/CustomSelect';
+import { validations, createTarget } from 'utils/constraints';
+// Imagenes
```

Comment:
Remember to use English only. Also, there's no need to separate imports with comments unless there are too many.

Reply:
It´s done!